apps/vr-tests/.storybook/preview.js

```diff
@@ -1,50 +1,8 @@
 // @ts-check
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31941&commitId=24f04e47388ae6e06e9db0338ba0a5ae72f14a43&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Compact Personas.default.chromium.png|0| Added|
|ActivityItem.CompactPersonas.default.chromium.png|0| Removed|

</details>


  <details><summary><b>Button (compat)</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Action Button Checked.default.chromium.png|0| Added|
|Button (compat).Action Button Checked.hover.chromium.png|0| Added|
|Button (compat).Action Button Checked.pressed.chromium.png|0| Added|
|Button (compat).Action Button Default - RTL.default.chromium.png|0| Added|
|Button (compat).Action Button Default - RTL.hover.chromium.png|0| Added|
|Button (compat).Action Button Default - RTL.pressed.chromium.png|0| Added|
|Button (compat).Action Button Default.default.chromium.png|0| Added|
|Button (compat).Action Button Default.hover.chromium.png|0| Added|
|Button (compat).Action Button Default.pressed.chromium.png|0| Added|
|Button (compat).Action Button Disabled.default.chromium.png|0| Added|
|Button (compat).Action Button Disabled.hover.chromium.png|0| Added|
|Button (compat).Action Button Disabled.pressed.chromium.png|0| Added|
|Button (compat).Action Button Icon Only.default.chromium.png|0| Added|
|Button (compat).Action Button Icon Only.hover.chromium.png|0| Added|
|Button (compat).Action Button Icon Only.pressed.chromium.png|0| Added|
|Button (compat).Action Button No Icon.default.chromium.png|0| Added|
|Button (compat).Action Button No Icon.hover.chromium.png|0| Added|
|Button (compat).Action Button No Icon.pressed.chromium.png|0| Added|
|Button (compat).no flex shrink.default.chromium.png|0| Added|
|Button (compat).primary with placeholder.default.chromium.png|0| Added|

</details>


  <details><summary><b>Button Command (compat)</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Command (compat).Default - RTL.default.chromium.png|0| Added|
|Button Command (compat).Default - RTL.hover.chromium.png|0| Added|
|Button Command (compat).Default - RTL.open.chromium.png|0| Added|
|Button Command (compat).Default - RTL.pressed.chromium.png|0| Added|
|Button Command (compat).Default.default.chromium.png|0| Added|
|Button Command (compat).Default.hover.chromium.png|0| Added|
|Button Command (compat).Default.open.chromium.png|0| Added|
|Button Command (compat).Default.pressed.chromium.png|0| Added|
|Button Command (compat).Root - RTL.default.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.open.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Command (compat).Root.default.chromium.png|0| Removed|
|Button Command (compat).Root.hover.chromium.png|0| Removed|
|Button Command (compat).Root.open.chromium.png|0| Removed|
|Button Command (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Compound (compat)</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound (compat).Compound Button Default - RTL.default.chromium.png|0| Added|
|Button Compound (compat).Compound Button Default - RTL.hover.chromium.png|0| Added|
|Button Compound (compat).Compound Button Default - RTL.pressed.chromium.png|0| Added|
|Button Compound (compat).Compound Button Default.default.chromium.png|0| Added|
|Button Compound (compat).Compound Button Default.hover.chromium.png|0| Added|
|Button Compound (compat).Compound Button Default.pressed.chromium.png|0| Added|
|Button Compound (compat).Root - RTL.default.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.hover.chromium.png|0| Removed|
|Button Compound (compat).Root - RTL.pressed.chromium.png|0| Removed|
|Button Compound (compat).Root.default.chromium.png|0| Removed|
|Button Compound (compat).Root.hover.chromium.png|0| Removed|
|Button Compound (compat).Root.pressed.chromium.png|0| Removed|

</details>


  <details><summary><b>Calendar</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar.Show Month as Overlay and no Go To Today.default.chromium.png|0| Added|
|Calendar.Show Month selection type.default.chromium.png|0| Added|
|Calendar.Show Week selection type.default.chromium.png|0| Added|
|Calendar.Show Work Week selection type.default.chromium.png|0| Added|

</details>


  <details><summary><b>Callout</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom center.default.chromium.png|2397| Changed|
|Callout.Beak 25.default.chromium.png|3298| Changed|
|Callout.Bottom right edge - RTL.default.chromium.png|3334| Changed|
|Callout.Left bottom edge.default.chromium.png|3449| Changed|
|Callout.Bottom auto edge.default.chromium.png|3351| Changed|
|Callout.No callout width specified.default.chromium.png|3342| Changed|
|Callout.No beak.default.chromium.png|3294| Changed|
|Callout.Left center.default.chromium.png|3227| Changed|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|1332| Changed|
|Callout.Bottom left edge.default.chromium.png|3298| Changed|
|Callout.Bottom right edge.default.chromium.png|3464| Changed|
|Callout.Gap space 25.default.chromium.png|3298| Changed|
|Callout.Bottom left edge - RTL.default.chromium.png|3284| Changed|
|Callout.Right bottom edge.default.chromium.png|3343| Changed|
|Callout.Left top edge.default.chromium.png|3046| Changed|
|Callout.Right center.default.chromium.png|3047| Changed|
|Callout.Root.default.chromium.png|3298| Changed|
|Callout.Right top edge.default.chromium.png|3331| Changed|
|Callout.Top auto edge.default.chromium.png|3268| Changed|
|Callout.Top left edge.default.chromium.png|3268| Changed|
|Callout.Top center.default.chromium.png|2392| Changed|
|Callout.Top right edge.default.chromium.png|3447| Changed|

</details>


  <details><summary><b>Checkbox</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Controlled Indeterminate checkbox without onChange callback.clicked.chromium.png|0| Added|
|Checkbox.Controlled Indeterminate checkbox without onChange callback.default.chromium.png|0| Added|
|Checkbox.Controlled Indeterminate checkbox without onChange callback.hover.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate checkbox.clicked.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate checkbox.default.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate checkbox.hover.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate defaultChecked checkbox.clicked.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate defaultChecked checkbox.default.chromium.png|0| Added|
|Checkbox.Uncontrolled Indeterminate defaultChecked checkbox.hover.chromium.png|0| Added|
|Checkbox.Uncontrolled and Controlled Indeterminate checkbox.clicked.chromium.png|0| Added|
|Checkbox.Uncontrolled and Controlled Indeterminate checkbox.default.chromium.png|0| Added|
|Checkbox.Uncontrolled and Controlled Indeterminate checkbox.hover.chromium.png|0| Added|

</details>


  <details><summary><b>ContextualMenu</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With secondaryText - RTL.click.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.default.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker.Show Month as Overlay and no Go To Today.click.chromium.png|0| Added|
|DatePicker.Show Month as Overlay and no Go To Today.default.chromium.png|0| Added|
|DatePicker.Show Month as Overlay and no Go To Today.hover datepicker.chromium.png|0| Added|
|DatePicker.Show Month as Overlay and no Go To Today.hover day.chromium.png|0| Added|
|DatePicker.With Label.click.chromium.png|0| Added|
|DatePicker.With Label.default.chromium.png|0| Added|
|DatePicker.With Label.hover datepicker.chromium.png|0| Added|
|DatePicker.Without Label.click.chromium.png|0| Added|
|DatePicker.Without Label.default.chromium.png|0| Added|
|DatePicker.Without Label.hover datepicker.chromium.png|0| Added|
|DatePicker.Without Value.click.chromium.png|0| Added|
|DatePicker.Without Value.default.chromium.png|0| Added|
|DatePicker.Without Value.hover datepicker.chromium.png|0| Added|

</details>


  <details><summary><b>DocumentCard</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.Compact with preview image.default.chromium.png|181548| Changed|
|DocumentCard.Compact with preview list.default.chromium.png|201654| Changed|

</details>


  <details><summary><b>Dropdown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Disabled.click.chromium.png|0| Added|
|Dropdown.Disabled.default.chromium.png|0| Added|
|Dropdown.Disabled.hover.chromium.png|0| Added|

</details>


  <details><summary><b>Fonts</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Fonts.Weights.default.chromium.png|479| Changed|

</details>


  <details><summary><b>IconButton (compat)</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|IconButton (compat).icon button with menu.hover icon.chromium.png|0| Added|
|IconButton (compat).icon button with menu.icon.chromium.png|0| Added|
|IconButton (compat).normal icon button.hover icon.chromium.png|0| Added|
|IconButton (compat).normal icon button.icon.chromium.png|0| Added|

</details>


  <details><summary><b>Keytip</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Root.default.chromium.png|189| Changed|
|Keytip.Offset.default.chromium.png|198| Changed|
|Keytip.Disabled.default.chromium.png|131| Changed|

</details>


  <details><summary><b>List</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.Root - RTL.default.chromium.png|12673| Changed|
|List.Root.default.chromium.png|12715| Changed|

</details>


  <details><summary><b>OverflowSet</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Vertical Direction.default.chromium.png|0| Added|

</details>


  <details><summary><b>Overlay</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Overlay.Dark.default.chromium.png|45| Changed|
|Overlay.Root.default.chromium.png|45| Changed|
|Overlay.Root - RTL.default.chromium.png|45| Changed|

</details>


  <details><summary><b>PeoplePicker</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal with placeholder.default.chromium.png|0| Added|
|PeoplePicker.Normal with placeholder.suggestions.chromium.png|0| Added|
|PeoplePicker.Normal with placeholder.chromium.png|0| Removed|

</details>


  <details><summary><b>Pivot</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Count and icon - RTL.click.chromium.png|70795| Changed|
|Pivot.Count and icon - RTL.hover-selected.chromium.png|70784| Changed|
|Pivot.Large.hover-selected.chromium.png|37685| Changed|
|Pivot.Count and icon - RTL.default.chromium.png|70784| Changed|
|Pivot.Count and icon.default.chromium.png|70832| Changed|
|Pivot.Large.default.chromium.png|37685| Changed|
|Pivot.Large.click.chromium.png|37687| Changed|
|Pivot.Root.hover-selected.chromium.png|31610| Changed|
|Pivot.Count and icon.hover-selected.chromium.png|70832| Changed|
|Pivot.Count and icon - RTL.hover.chromium.png|70784| Changed|
|Pivot.Tabs - RTL.default.chromium.png|20477| Changed|
|Pivot.Tabs - RTL.click.chromium.png|20489| Changed|
|Pivot.Tabs - RTL.hover.chromium.png|20477| Changed|
|Pivot.Tabs large.default.chromium.png|23190| Changed|
|Pivot.Count and icon.hover.chromium.png|70832| Changed|
|Pivot.Root.default.chromium.png|31610| Changed|
|Pivot.Root.hover.chromium.png|31610| Changed|
|Pivot.Tabs.click.chromium.png|20527| Changed|
|Pivot.Tabs large.hover.chromium.png|23190| Changed|
|Pivot.Root.click.chromium.png|31612| Changed|
|Pivot.Tabs large.click.chromium.png|23192| Changed|
|Pivot.Tabs large.hover-selected.chromium.png|23190| Changed|
|Pivot.Tabs.hover.chromium.png|20525| Changed|
|Pivot.Large.hover.chromium.png|37685| Changed|
|Pivot.Tabs.default.chromium.png|20525| Changed|
|Pivot.Tabs.hover-selected.chromium.png|20525| Changed|
|Pivot.Tabs - RTL.hover-selected.chromium.png|20477| Changed|
|Pivot.Count and icon.click.chromium.png|70834| Changed|
|Pivot.Overflow.chromium.png|0| Added|
|Pivot.Tabs - RTL.chromium.png|0| Added|

</details>


  <details><summary><b>Shimmer</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shimmer.Data loaded.default.chromium.png|48| Changed|
|Shimmer.50% width.default.chromium.png|80| Changed|
|Shimmer.Data not loaded.default.chromium.png|160| Changed|
|Shimmer.Circle Gap Line.default.chromium.png|163| Changed|
|Shimmer.Basic.default.chromium.png|160| Changed|
|Shimmer.Circle Gap Line - RTL.default.chromium.png|163| Changed|

</details>


  <details><summary><b>SpinButton</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton.Label on top with icon.default.chromium.png|26614| Changed|
|SpinButton.Label on top.default.chromium.png|24531| Changed|

</details>


  <details><summary><b>TagPicker</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.With long tag.default.chromium.png|9802| Changed|
|TagPicker.TagPicker disabled.chromium.png|7| Changed|
|TagPicker.Default.Open Suggestion Menu.chromium.png|0| Added|
|TagPicker.Default.Suggestion Menu Item Hover.chromium.png|0| Added|
|TagPicker.Default.default.chromium.png|0| Added|
|TagPicker.Tag Item Selected - RTL.Tag Item Clear Button Hover.chromium.png|0| Added|
|TagPicker.Tag Item Selected - RTL.Tag Item Clear Button Selected Hover.chromium.png|0| Added|
|TagPicker.Tag Item Selected - RTL.Tag Item Hover.chromium.png|0| Added|
|TagPicker.Tag Item Selected - RTL.Tag Item Select.chromium.png|0| Added|
|TagPicker.Tag Item Selected - RTL.default.chromium.png|0| Added|
|TagPicker.Tag Item Selected.Tag Item Clear Button Hover.chromium.png|0| Added|
|TagPicker.Tag Item Selected.Tag Item Clear Button Selected Hover.chromium.png|0| Added|
|TagPicker.Tag Item Selected.Tag Item Hover.chromium.png|0| Added|
|TagPicker.Tag Item Selected.Tag Item Select.chromium.png|0| Added|
|TagPicker.Tag Item Selected.default.chromium.png|0| Added|
|TagPicker.Root.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Root.default.chromium.png|0| Removed|

</details>


  <details><summary><b>TextField</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Error1 - RTL.click.chromium.png|0| Added|
|TextField.Error1 - RTL.default.chromium.png|0| Added|
|TextField.Error1 - RTL.hover.chromium.png|0| Added|
|TextField.Error - RTL.click.chromium.png|0| Removed|
|TextField.Error - RTL.default.chromium.png|0| Removed|
|TextField.Error - RTL.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.default.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.default.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>ThemeProvider</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider.Customizer wraps ThemeProvider.default.chromium.png|0| Added|
|ThemeProvider.ThemeProvider wraps Customizer.default.chromium.png|0| Added|
|ThemeProvider.Use contextual theme over global theme if defined.default.chromium.png|0| Added|
|ThemeProvider.Use contextual theme over global theme if defined.global theme changed.chromium.png|0| Added|
|ThemeProvider.Use updated global theme.default.chromium.png|0| Added|
|ThemeProvider.Use updated global theme.global theme changed.chromium.png|0| Added|

</details>


  <details><summary><b>Tile</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tile.Media tile with nameplate hidden until hover.default.chromium.png|0| Added|
|Tile.Media tile with nameplate hidden until hover.hover.chromium.png|0| Added|
|Tile.Media tile with single activity line.default.chromium.png|0| Added|
|Tile.Media tile with single activity line.hover.chromium.png|0| Added|
|Tile.Media tile with two activity lines.default.chromium.png|0| Added|
|Tile.Media tile with two activity lines.hover.chromium.png|0| Added|
|Tile.Media tile with very long name and activity.default.chromium.png|0| Added|
|Tile.Media tile with very long name and activity.hover.chromium.png|0| Added|

</details>


  <details><summary><b>TilesList</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TilesList.Basic - RTL.default.chromium.png|2538| Changed|
|TilesList.Basic.default.chromium.png|2538| Changed|

</details>


  <details><summary><b>Tooltip</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.default.chromium.png|609| Changed|
|Tooltip.Default.chromium.png|302| Changed|
|Tooltip.Two Tooltips.chromium.png|0| Added|
|Tooltip.Two Tooltips.hover inner.chromium.png|0| Added|
|Tooltip.Two Tooltips.hover outer.chromium.png|0| Added|

</details>


  <details><summary><b>react-charting-AreaChart</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Basic - Dark Mode.default.chromium.png|18234| Changed|
|react-charting-AreaChart.Basic - Dark Mode.hover.chromium.png|18234| Changed|
|react-charting-AreaChart.Custom Accessibility - Dark Mode.default.chromium.png|18234| Changed|
|react-charting-AreaChart.Multiple - Dark Mode.default.chromium.png|18234| Changed|
|react-charting-AreaChart.Multiple - Dark Mode.hover.chromium.png|17458| Changed|

</details>


  <details><summary><b>react-charting-DonutChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-DonutChart.Basic - Dark Mode.default.chromium.png|8152| Changed|
|react-charting-DonutChart.Dynamic - Dark Mode.default.chromium.png|8509| Changed|

</details>


  <details><summary><b>react-charting-GaugeChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-GaugeChart.Variant - Dark Mode.default.chromium.png|7955| Changed|
|react-charting-GaugeChart.Basic - Dark Mode.default.chromium.png|7866| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic - Dark Mode.default.chromium.png|14658| Changed|

</details>


  <details><summary><b>react-charting-HorizontalBarChart</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.Basic - Dark Mode.default.chromium.png|17948| Changed|
|react-charting-HorizontalBarChart.Variant - Dark Mode.default.chromium.png|14015| Changed|
|react-charting-HorizontalBarChart.Basic - Dark Mode.hover.chromium.png|17947| Changed|
|react-charting-HorizontalBarChart.With_Benchmark - Dark Mode.default.chromium.png|14587| Changed|
|react-charting-HorizontalBarChart.With_Axis - Dark Mode.default.chromium.png|18949| Changed|

</details>


  <details><summary><b>react-charting-Legend</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-Legend.Basic - Dark Mode.default.chromium.png|8652| Changed|
|react-charting-Legend.Overflow - Dark Mode.expanded.chromium.png|18949| Changed|
|react-charting-Legend.Overflow - Dark Mode.default.chromium.png|18949| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Basic - Dark Mode.hover.chromium.png|0| Removed|
|react-charting-LineChart.Basic - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Events - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Events - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Gaps - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Gaps - RTL.default.chromium.png|0| Removed|
|react-charting-LineChart.Multiple - Dark Mode.default.chromium.png|0| Removed|
|react-charting-LineChart.Multiple - RTL.default.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-MultiStackBarChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-MultiStackBarChart.With_Placeholders - Dark Mode.default.chromium.png|13854| Changed|
|react-charting-MultiStackBarChart.Basic_Absolute - Dark Mode.default.chromium.png|18466| Changed|
|react-charting-MultiStackBarChart.Basic_PartToWhole - Dark Mode.default.chromium.png|18198| Changed|

</details>


  <details><summary><b>react-charting-PieChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic - RTL.default.chromium.png|88| Changed|
|react-charting-PieChart.Basic.default.chromium.png|88| Changed|
|react-charting-PieChart.Basic - Dark Mode.default.chromium.png|29192| Changed|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.PlaceHolder - Dark Mode.default.chromium.png|23811| Changed|
|react-charting-SankeyChart.Basic - Dark Mode.default.chromium.png|24026| Changed|

</details>


  <details><summary><b>react-charting-SparkLineChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|7445| Changed|
|react-charting-SparkLineChart.Basic - Dark Mode.default.chromium.png|9510| Changed|
|react-charting-SparkLineChart.Basic.default.chromium.png|7393| Changed|

</details>


  <details><summary><b>react-charting-StackedBarChart</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-StackedBarChart.Benchmark - Dark Mode.default.chromium.png|8563| Changed|
|react-charting-StackedBarChart.Basic.default.chromium.png|2591| Changed|
|react-charting-StackedBarChart.Benchmark.default.chromium.png|556| Changed|
|react-charting-StackedBarChart.Benchmark - RTL.default.chromium.png|556| Changed|
|react-charting-StackedBarChart.Basic - Dark Mode.default.chromium.png|18824| Changed|
|react-charting-StackedBarChart.Basic - RTL.default.chromium.png|2591| Changed|
|react-charting-StackedBarChart.Multiple - Dark Mode.default.chromium.png|19771| Changed|

</details>


  <details><summary><b>react-charting-TreeChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|38148| Changed|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|38148| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis - Dark Mode.default.chromium.png|18234| Changed|
|react-charting-VerticalBarChart.Date Axis- VBC - Dark Mode.default.chromium.png|20915| Changed|
|react-charting-VerticalBarChart.Stacked - Dark Mode.default.chromium.png|18234| Changed|
|react-charting-VerticalBarChart.Grouped - Wrap Labels - Dark Mode.default.chromium.png|20021| Changed|
|react-charting-VerticalBarChart.Stacked - Date Axis - Dark Mode.default.chromium.png|18234| Changed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends - Dark Mode.default.chromium.png|20915| Changed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - Dark Mode.default.chromium.png|0| Removed|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - RTL.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
some of these diffs are actually valid, wondering if that's expected or something additional is missing as part of this refactoring ? 

example of Pie Chart:

<img width="1446" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/6b2855d9-e560-4143-a288-f728e31b2a41">